src/arch/riscv/rv32/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "architecture"
+name = "arch"
```

Comment:
are you sure this will work? It was archi at some point, we originally changed it due to a name conflict.

Reply:
It seems to work. Lets see if it passes the tests.